diff -ur perl-5.28.2/hints/linux.sh perl-5.28.2.patched/hints/linux.sh
--- perl-5.28.2/hints/linux.sh	2019-04-02 15:36:35.000000000 -0500
+++ perl-5.28.2.patched/hints/linux.sh	2020-04-14 10:46:50.660416347 -0500
@@ -165,6 +165,9 @@
 # plibpth to bypass this check.
 if [ -x /usr/bin/gcc ] ; then
     gcc=/usr/bin/gcc
+# clang also provides -print-search-dirs
+elif ${cc:-cc} --version 2>/dev/null | grep -q '^clang ' ; then
+    gcc=${cc:-cc}
 else
     gcc=gcc
 fi
Only in perl-5.28.2.patched: patchlevel.bak
diff -ur perl-5.28.2/patchlevel.h perl-5.28.2.patched/patchlevel.h
--- perl-5.28.2/patchlevel.h	2019-04-05 14:49:11.000000000 -0500
+++ perl-5.28.2.patched/patchlevel.h	2020-04-14 10:46:50.660416347 -0500
@@ -137,6 +137,7 @@
 	,"uncommitted-changes"
 #endif
 	PERL_GIT_UNPUSHED_COMMITS    	/* do not remove this line */
+	,"Devel::PatchPerl 1.90"
 	,NULL
 };
 
