diff -uNr apache_1.3.41.orig/src/os/unix/os.h apache_1.3.41/src/os/unix/os.h
--- apache_1.3.41.orig/src/os/unix/os.h	2006-07-12 03:16:05.000000000 -0500
+++ apache_1.3.41/src/os/unix/os.h	2016-07-06 08:27:21.394527656 -0500
@@ -33,7 +33,11 @@
 /* Compiler supports inline, so include the inlineable functions as
  * part of the header
  */
+#ifdef __GNUC_STDC_INLINE__ 
+#define INLINE extern ap_inline __attribute__((gnu_inline)) 
+#else 
 #define INLINE extern ap_inline
+#endif
 
 INLINE int ap_os_is_path_absolute(const char *file);
 
